src/Advection/tracer_advection_operators.jl

```diff
@@ -1,6 +1,34 @@
 using Oceananigans.Operators: Vᶜᶜᶜ
 using Oceananigans.Fields: ZeroField
 
+struct TracerAdvection{N, FT, A, B, C} <: AbstractAdvectionScheme{N, FT}
+    x :: A
+    y :: B
+    z :: C
+
+    TracerAdvection{N, FT}(x::A, y::B, z::C) where {N, FT, A, B, C} = new{N, FT, A, B, C}(x, y, z)
+end
+
+"""
+    function TracerAdvection(; x, y, z)
+
+builds a `TracerAdvection` type with different reconstructions in `x`, `y`, and `z`
```

Comment:
This is wrong --- I don't think there is any enforcement of having different reconstructions. The description should simply mention that `x, y, z` represent reconstruction in the `x, y, z` direction.